server/src/main/scala/org/http4s/server/middleware/CORS.scala

```diff
@@ -360,7 +360,7 @@ sealed class CORSPolicy(
     val varyHeaderNonOptions =
       allowOrigin match {
         case AllowOrigin.Match(_) =>
-          Header.Raw(ci"Vary", Header[Origin].name.toString).some
+          Vary(Header[Origin].name).some
```

Comment:
_[compile-time micro-opt nitpick]_: we can override the `HeaderCompanion#name` to be a `val` and use it when needed